ros2/images.yaml.em

```diff
@@ -23,6 +23,8 @@ images:
             - python3-pip
             - python3-setuptools
             - python3-vcstool
+            - libtinyxml-dev
```

Comment:
Yes. 
- `libeigen3-dev` is needed by `orocos_kdl` package.
- `libasio-dev` and `libtinyxml2-dev` are needed by `Fast-RTPS`.
- `flake8` and `flake8-import-order` are needed by `ament_cmake_flake8`